WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderBlogActions.java

```diff
@@ -505,87 +505,4 @@ public void onErrorResponse(VolleyError volleyError) {
         String path = "me/block/sites/" + Long.toString(blockResult.blogId) + "/delete";
         WordPress.getRestClientUtilsV1_1().post(path, listener, errorListener);
     }
-
-    /**
-     * Updates the imageUrl for a feed. Since the /read/feed/{feedId} endpoint does not have this data,
-     * the request is made to the /read/sites/{blogId} endpoint.
-     * @param blogId
-     * @param blogUrl
-     * @param feedId
-     * @param infoListener
-     */
-    public static void updateImageForFeedByBlog(long blogId,
-                                      final String blogUrl,
```

Comment:
This was code was added in the first PR and can safely be removed because it is not used.